lfs.c

```diff
@@ -1525,7 +1525,7 @@ static int lfs_dir_compact(lfs_t *lfs,
         if (lfs_pair_cmp(dir->pair, (const lfs_block_t[2]){0, 1}) == 0) {
             // oh no! we're writing too much to the superblock,
             // should we expand?
-            lfs_ssize_t res = lfs_fs_size(lfs);
+            lfs_ssize_t res = _lfs_fs_size(lfs);
```

Comment:
I was thinking a good name for these internal functions may be "lfs_fs_size_raw", "lfs_dir_read_raw", etc. Thoughts?